docs/guides/deploy-guide/bootstrap.md

```diff
@@ -22,78 +22,131 @@ The guide for this can be found in the section [Provisioning of bare-metal nodes
 The following steps are applied to bootstrap all nodes. After the completion of the bootstrap,
 the nodes are already ready for use.
 
-1. Create operator user.
 
-   ```
-   osism apply operator -u osism
-   ```
+## Group assignments
 
-   * When using the [osism/node-image](https://github.com/osism/node-image) the user is `osism`
-     and the password of this user is `password`. If you install Ubuntu manually the user usually
-     is `ubuntu`. If you want to use any other user here, that's no problem. It is important that
-     this user has sudo rights. The password according to what you have set yourself.
+Assign the nodes to the suitable roles in `inventory/20-roles`
 
-   * The operator public SSH key has to be added in advance on all nodes to `authorized_keys` file
-     of the user specified with `-u`. This key is stored as `operator_public_key` in the file
-     `environments/configuration.yml`.
+* Assign all nodes to the "generic" group
+* Assign the nodes to the suitable groups
 
-     Alternatively (not recommended), the password can be stored in plain text in a file `/opt/configuration/secrets/conn_password`.
-     The parameter `--conn-pass-file /opt/configuration/secrets/conn_password` must then also be specified:
 
-     ```
-     osism apply operator -u osism \
-       --conn-pass-file /opt/configuration/secrets/conn_password
-     ```
+TODO: Does it make sense to describe the purpose of the groups with a few words to
+help newbies to make the right descicions?
 
-   * It is important that this user has sudo rights with `NOPASSWD`.
+| Groupname             | Description                                           |
+|-----------------------|-------------------------------------------------------|
+| `[generic]`           | Roles which do not have any service specific settings |
+| `[manager]`           | Roles for the manager nodes                           |
+| `[monitoring]`        | The log, metric and alerting infrastructure roles     |
+| `[control]`           | Openstack contoller roles                             |
+| `[compute]`           | Openstack compute nodes roles                         |
+| `[ceph-control]`      | Ceph mon servers                                      |
+| `[ceph-resource]`     | Ceph OSDs, ...                                        |
+| `[ceph-rgw:children]` | Rados Gateway roles                                   |
```

Comment:
Does it make sense to describe the purpose of the groups with a few words to
help newbies to make the right decisions?